src/core/memory.h

```diff
@@ -2,3 +2,10 @@ int      lock_memory();
 void    *allocate_huge_page(int size);
 uint64_t phys_page(uint64_t virt_page);
 
+void setup_signal();
+
+
+struct map_ids_t {
+   int huge_page_bits;
+   int ids[16384];
```

Comment:
I had to increase this number on my setup, otherwise I was getting:
```
snabb: core/memory.c:104: allocate_huge_page: Assertion `offst < (sizeof(map_ids->ids) / sizeof(int))' failed.
```
when inspected the `offst` value was int the range `17001` to `170013`.